tests/hipscat_import/test_map_reduce.py

```diff
@@ -149,12 +147,47 @@ def test_map_small_sky_order0(small_sky_single_file):
         npt.assert_array_equal(result, expected)
         assert (result == expected).all()
 
-        file_name = os.path.join(tmp_dir, "pixel_11", "shard_0.parquet")
+        file_name = os.path.join(tmp_dir, "pixel_11", "shard_0_0.parquet")
         expected_ids = [*range(700, 831)]
         ft.assert_parquet_file_ids(file_name, "id", expected_ids)
 
 
-@pytest.mark.filterwarnings("ignore::DeprecationWarning")
+def test_map_small_sky_chunk1(small_sky_single_file):
+    """Test loading the small sky catalog and partitioning each object one row at a time"""
+
+    def csv_single_line_reader(input_file):
+        with pd.read_csv(input_file, chunksize=1) as reader:
+            for chunk in reader:
+                chunk.reset_index(inplace=True)
+                yield chunk
```

Comment:
Another instance where a reader class could have all this logic with a chunksize argument instead of writing out the function